build/commands/lib/guessConfig.ts

```diff
@@ -4,7 +4,7 @@
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
 import config from './config'
-import path from 'path'
+import path from 'path/posix'
```

Comment:
what was this needed for?

Reply:
Without this it can't find the build output directory on Windows.

Reply:
huh odd!